db/migrate/20200212171537_remove_ovirt_sdk4_from_event_stream.rb

```diff
@@ -0,0 +1,27 @@
+class RemoveOvirtSdk4FromEventStream < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+  include MigrationHelper
+
+  class EventStream < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time("Removing OvirtSDK4 types from EmsEvents") do
+      base_relation = EventStream.in_my_region.where(:source => "RHEVM")
+        .where("full_data LIKE ?", "%object:OvirtSDK4::Event%")
+
+      say_batch_started(base_relation.size)
+
+      loop do
+        count = base_relation.limit(50_000)
+          .update_all("full_data = REGEXP_REPLACE(full_data, '!ruby/object:OvirtSDK4::\\w+', '!ruby/object:Hash', 'g')")
```

Comment:
Are the objects always directly under the OvirtSDK namespace?  That is, could there ever be more than one set of `::`

Reply:
Doesn't look like it:
```
>> OvirtSDK4::Identified.descendants.map(&:name).any? { |n| n.scan("::").count > 1 }
=> false
```

(`OvirtSDK4::Identified` is the base class that all others are derived from)